### DISCUSSION TITLE
Asking for joining team

### DISCUSSION THREAD

**Discussion by WAR-DEPT** [ROM-builders/temporary#23559] [General]
Device tree -- 
https://github.com/WAR-DEPT/device_xiaomi_miatoll.git
https://github.com/WAR-DEPT/device_xiaomi_sm6250-common.git
https://github.com/WAR-DEPT/vendor_xiaomi_miatoll.git
https://github.com/WAR-DEPT/vendor_xiaomi_sm6250-common.git

Successful build-
https://cirrus-ci.com/task/4920507330461696?logs=build#L21

Manifest-
https://github.com/WAR-DEPT/local_manifest.git

Tg- @HarjeetSingh46
GitHub- https://github.com/WAR-DEPT

thank you for providing ROM-builders,
Have a great day 🙏😊

**Comment by ghost** [comment]
Your commit messages are not proper. Please commit changes with a proper commit message. Plus you also need to learn cherry-picking.

**Comment by Apon77** [comment]
please follow #learn instruction from telegram group
